### PR TITLE
Feat/contributors guideline

### PR DESCRIPTION
**Pull request by ocontant**
Draft of contributing guide.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+## To contribute to this project, please adhere to the following guidelines:
+
+### 1. Git Workflow
+
+We use a GitFlow workflow to manage changes to this project's codebase. Follow these general steps when contributing code:
+
+1. Fork the repository and create a new branch with a descriptive kebab-case name (e.g., `new-feature-x`).
+2. Implement your changes and ensure that all tests pass.
+3. Commit your changes using conventional commit messages with GitFlow flags. Begin the commit message with a tag indicating the change type, such as "feat" (new feature), "fix" (bug fix), "docs" (documentation), or "refactor" (code refactoring), followed by a brief summary of the changes (e.g., `feat: Add new feature X to the project`).
+4. Submit a pull request with a clear and concise description of your changes and the reasons behind them.
```
Comment (danorlando) [inline CONTRIBUTING.md:33]: Also include steps to test the PR. "Submit a pull request with a clear and concise description of your changes and the reasons behind them. Be sure to include the steps to test the PR.

**Review by danorlando — CHANGES_REQUESTED** [review]
This is perfect. Just need to add the one additional sentence about including the steps to test the PR and then we're good to go!

File: CONTRIBUTING.md (hunk 1 above)
Comment (fuegovic) [reply CONTRIBUTING.md:33]: For reference, I've put a big note in the top of the PR section in the version I have (and I also have a doc specifically about testing procedure)

```
## 3. Pull Request Process


### Note: Submit a pull request with a clear and concise description of your changes and the reasons behind them. Be sure to include the steps to test the PR.

1. Ensure any install or build dependencies are removed before the end of the layer when doing a
   build.
2. Update the README.md with details of changes to the interface, this includes new environment
   variables, exposed ports, useful file locations and container parameters.
3. Increase the version numbers in any examples files and the README.md to the new version that this
   Pull Request would represent. The versioning scheme we use is [SemVer](http://semver.org/).

Ensure that your changes meet the following criteria when submitting a pull request:
- All tests pass.
- The code is well-formatted and adheres to our coding standards.
- The commit history is clean and easy to follow. (Use Squash to clean your commit history)
- The pull request description clearly outlines the changes and the reasons behind them.
```

Comment (fuegovic) [reply CONTRIBUTING.md:33]: @danorlando , better check the updated one from the new docs structure to see if it's ok. I updated it like previously shown⬆️. If there are changes to be made, they need to be done from the updated branch to not break anything

**Comment by danorlando** [comment]
We can close this PR since the contents of this file was added to the new document structure by @fuegovic

**Pull request closed without merging**